openedx/core/djangoapps/credit/exceptions.py

```diff
@@ -0,0 +1,6 @@
+""" This module contains the exceptions raised in credit course requirements """
+
+
+class InvalidCreditRequirements(Exception):
+    """ Contains exceptions related to credit module """
```

Comment:
The exception occurs when the `requirements` dictionary is in an invalid format.  Please make this docstring more specific.